### PR TITLE
Hydroponics plumbing

### PR DESCRIPTION
**Pull request by SpaceCheetah**
# About The Pull Request
A few fixes for hydroponics tray plumbing.

## Why It's Good For The Game
Plumbing in general is unintuitive and somewhat broken. Fixing those issues helps makes the system usable. 

## A Port?
No.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
del: automated hydroponics tray remove_unfasten_wrench override (duplicate behavior, extra messages)
fix: prevent unanchored hydroponics trays from accepting plumbing reagents
fix: removed a use-after-delete caused by calling create_reagents twice in automated hydroponics trays, which broke plumbing
tweak: allow hydroponics trays to accept plumbing reagents from all directions
del: remove tray rotations (sprite didn't update anyway, and was only necessary for plumbing)
/:cl:

### CODE REVIEW

**Comment by SpaceCheetah** [comment]
Also, I initially discussed the issues I'm fixing here in https://discord.com/channels/908082639704064051/1139434393497239612/1139434393497239612.

File: code/datums/components/plumbing/_plumbing.dm [hunk 1]
```diff
@@ -213,3 +213,11 @@
 /datum/component/plumbing/tank
 	demand_connects = WEST
 	supply_connects = EAST
+
+/datum/component/plumbing/hydroponics
```
Comment (KeplerWasTaken) [inline code/datums/components/plumbing/_plumbing.dm:217]: Can this be modularized please?

File: code/datums/components/plumbing/_plumbing.dm [hunk 2]
```diff
@@ -213,3 +213,11 @@
...
+	demand_connects = NORTH | SOUTH | EAST | WEST
+	//hydroponics cuts the overlays anyway, no point drawing then erasing them
+	use_overlays = FALSE
+
+/datum/component/plumbing/hydroponics/Initialize()
```
Comment (KeplerWasTaken) [inline code/datums/components/plumbing/_plumbing.dm:222]: This as well please

File: code/datums/components/plumbing/_plumbing.dm (hunk 1 above)
Comment (SpaceCheetah) [reply code/datums/components/plumbing/_plumbing.dm]: Moved it to modular_splurt. Is that all you mean by that?

**Pull request merged**